src/server/models/index.js

```diff
@@ -61,6 +61,7 @@ const tableList = [
   "question_response",
   "tag",
   "tag_campaign_contact",
+  "owned_phone_number",
```

Comment:
I know it's silly, but can you create a models/owned-phone-number.js file in the style of the old ones.  even if it's dead code, I'd like us to keep that directory documenting the current schema for all the models (rather than needing to search through changes in the migrations).